ui/common/src/commonMain/kotlin/social/androiddev/common/composables/buttons/MastodonTextButton.kt

```diff
@@ -0,0 +1,156 @@
+package social.androiddev.common.composables.buttons
+
+//import androidx.compose.desktop.ui.tooling.preview.Preview
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.material.ButtonColors
+import androidx.compose.material.ContentAlpha
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.material.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.Immutable
+import androidx.compose.runtime.State
+import androidx.compose.runtime.rememberUpdatedState
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+import social.androiddev.common.theme.MastodonTheme
+
+@Composable
+fun MastodonTextButton(
+    text: String,
+    modifier: Modifier = Modifier,
+    enabled: Boolean = true,
+    colors: ButtonColors = textButtonColors(),
+    onClick: () -> Unit,
+) {
+    TextButton(
+        modifier = modifier,
+        enabled = enabled,
+        colors = colors,
+        onClick = onClick
+    ) {
+        Text(text.uppercase())
```

Comment:
Is this text always uppercase? maybe just put the text in here and the caller can decided what the text looks like?

Reply:
Yes, all the button composables (MastodonButton, MastodonOutlinedButton) and this force upper case so it's consistent across all buttons in the app. I wanted to initially make this as part of the button style but that's not supported. 

I don't feel strongly, I can remove, but jut wanted to have some sort of consistency to the button text styling so we don't have Sentence Case and then ALL CAPS in different places.